dd-sdk-android/src/test/kotlin/com/datadog/android/tracing/internal/net/TracesOkHttpUploaderTest.kt

```diff
@@ -43,6 +43,6 @@ internal class TracesOkHttpUploaderTest : DataOkHttpUploaderTest<TracesOkHttpUpl
     }
 
     override fun expectedPathRegex(): String {
-        return "\\/v1\\/input\\/$fakeToken$"
+        return "^\\/v1\\/input\\/$fakeToken$"
```

Comment:
So, adding `batch_time` doesn't solve the problem for the APM Intake, and this is why we mix both approaches (`?batch_time` with NTP time)?

Reply:
Yes